content/02kapitel.tex

```diff
@@ -39,5 +39,37 @@ \section{Rollenverteilung}
+
+    Zu Beginn jedes Sprints findet einmalig das \glqq Sprint Planning\grqq~statt. Dem Namen entsprechend wird in diesem Meeting ein genereller Plan 
+    für den Sprint erstellt. Dazu wird zuerst vom Product Owner ein Ziel für den Sprint vorgeschlagen und nach einer Diskussion im Team festgelegt.
+    Danach werden Tickets aus dem Backlog ausgewählt, welche für das Ziel relevant sind und innerhalb des Sprints bearbeitet werden sollen. Dabei 
+    wird auch eingeschätzt, wie viel in einem Sprint realistisch schaffbar ist. Zum einen werden dazu in \glqq Estimation-Meetings\grqq~die Tickets
+    bezüglich des jeweiligen Arbeitsaufwandes eingeschätzt und zum anderen gibt jedes Teammitglied eine Kapazität (in Story-Points) für den Sprint an.
+    Das Resultat eines \glqq Sprint Plannings\grqq~ist das Sprint-Ziel und der Sprint Backlog.
+
+    Um im Voraus Storys für das Sprint Planning zu erstellen und auszuplanen erfolgt das \enquote{Roadmap-Meeting}.
+    Hierbei wurden die Storys, welche vom PO erstellt wurden, mit dem Architekten abgesprochen.
+    Ziel ist es innerhalb dieses Meetings Storys auf ihre technische Umsetzbarkeit zu kontrollieren und möglicherweise in kleinere aufzuteilen.
+    Dieses Meeting fand meistens an den Montagen vor den jeweiligen Estimation Meetings statt.
+
+    Innerhalb der \glqq Estimation-Meetings\grqq~werden neue, meist vom Product Owner erstellte Tickets im Backlog von den jeweiligen Service-Leads
+    vorgestellt, deren Service vom Ticket betroffen ist (oder dem Scrum Master, sollte kein Service direkt betroffen sein). Nachdem Fragen zum Ticket
+    geklärt wurden, wird der Arbeitsaufwand des Tickets in Story-Points geschätzt, wobei Fibonacci-Zahlen von 1 bis 21 verwendet werden. Zum Schätzen
+    gibt jedes Teammitglied eine Zahl an (oder enthält sich) und am Ende wird die Zahl mit den meisten Stimmen übernommen. 
+    Dieses Meeting findet im Anschluss eines Dailys oder BI-Weeklys statt, wenn seit dem Letzten neue Tickets hinzugefügt wurden.
+
+    In \glqq Dailys\grqq~wird von jedem Teammitglied der aktuelle Stand erläutert und Probleme, wie blockierende Tickets oder aufgetretene 
+    Unklarheiten angesprochen. Wenn nötig, kann daraufhin auch der Plan des aktuellen Sprints angepasst werden. Normalerweise wird dieses Meeting 
+    dem Namen entsprechend täglich abgehalten, ist jedoch in unserem Fall nur einmal wöchentlich im Rahmen der Vorlesung vorgesehen. Um einen 
+    häufigeren Austausch zu ermöglichen haben wir zusätzlich ein \glqq BI-Weekly\grqq~eingeführt, welches inhaltlich dem Daily gleicht und ebenso 
+    wöchentlich abgehalten wird.
+
+    Zum Abschluss eines Sprints finden \glqq Review\grqq~und danach \glqq Retro\grqq~statt. Im Review wird besprochen, was im Sprint erreicht 
+    wurde und wie dies den allgemeinen Fortschritt des Projektes beeinflusst. Dabei wird auch besprochen, was als Nächstes zu Erreichen ist. 
```

Comment:
Im Review wird den Stakeholdern den fortschritt gezeigt 

(Stakeholder in unserem fall der ehrenwerte garidis)